src/containers/ChainOverview/useFetchChainChartData.tsx

```diff
@@ -627,40 +627,6 @@ const formatBarChart = ({
 	hasNoPrice?: boolean
 	denominationPriceHistory: Record<string, number> | null
 }): Array<[number, number]> => {
-	if (['weekly', 'monthly', 'cumulative'].includes(groupBy)) {
-		const store = {}
-		let total = 0
-		const isWeekly = groupBy === 'weekly'
-		const isMonthly = groupBy === 'monthly'
-		const isCumulative = groupBy === 'cumulative'
-		for (const [date, value] of data) {
-			const dateKey = isWeekly
-				? lastDayOfWeek(dateInMs ? +date : +date * 1e3)
-				: isMonthly
-				? firstDayOfMonth(dateInMs ? +date : +date * 1e3)
-				: dateInMs
-				? +date / 1e3
-				: +date
-			// sum up values as it is bar chart
-			if (denominationPriceHistory) {
-				const price = denominationPriceHistory[String(dateInMs ? date : +date * 1e3)]
-				store[dateKey] = (store[dateKey] ?? 0) + (price ? value / price : 0) + total
-				if (isCumulative && price) {
-					total += value / price
-				}
-			} else {
-				store[dateKey] = (store[dateKey] ?? 0) + value + total
-				if (isCumulative) {
-					total += value
-				}
-			}
-		}
-		const finalChart = []
-		for (const date in store) {
-			finalChart.push([+date * 1e3, store[date]])
-		}
-		return finalChart
-	}
```

Comment:
grouping by monthly breaks on home page if you remove this code
<img width="832" height="544" alt="Screenshot 2" src="https://github.com/user-attachments/assets/4eebdd47-9059-4589-9b9b-74515a5a8bd6" />